boards/common/pyocd.board.cmake

```diff
@@ -2,4 +2,5 @@
 
 board_set_flasher_ifnset(pyocd)
 board_set_debugger_ifnset(pyocd)
+board_set_reset_ifnset(pyocd)
```

Comment:
For consistency with the other `board_set_<foo>er_ifnset` functions, I think this should be named `board_set_resetter_ifnset()`.